### ISSUE TITLE
'nu script.nu' ineligible in the terminal with light background 

### ISSUE THREAD

**Issue by agavrilov** [nushell/nushell#12498]
### Describe the bug

I have PowerShell with a light background. When I run 'nu script.ru', the output of the script is shown in the white color which is not visible on the white background of the terminal.

### How to reproduce

1. Use light background on the terminal
2. Save the table [[c1 c2]; [t1 t2]] to test.nu 
3. Run nu test.nu    
4. Observe the output is shown in white colors

### Expected behavior

The output of tables from scripts should be visible

### Screenshots

![image](https://github.com/nushell/nushell/assets/3209180/c067b165-d202-4bce-b14b-8f2b112214b8)

### Configuration

| key                | value                                    |
| ------------------ | ---------------------------------------- |
| version            | 0.92.2                                   |
| branch             |                                          |
| commit_hash        | 2a08a18b26865a86c793d183b9b042220ecb733a |
| build_os           | windows-x86_64                           |
| build_target       | x86_64-pc-windows-msvc                   |
| rust_version       | rustc 1.77.2 (25ef9e3d8 2024-04-09)      |
| rust_channel       | 1.77.2-x86_64-pc-windows-msvc            |
| cargo_version      | cargo 1.77.2 (e52e36006 2024-03-26)      |
| build_time         | 2024-04-10 21:26:02 +00:00               |
| build_rust_channel | release                                  |
| allocator          | mimalloc                                 |
| features           | dataframe, default, sqlite, trash, which |
| installed_plugins  |                                          |

### Additional context

Related issues are #12490, #5961 

Please, do not close the issue and ask to re-open it if I have any questions! Github doesn't allow to reopen the issue if I'm not a contributor to the repository!

**Comment by fdncred** [comment]
Didn't we already discuss this? I distinctly remember saying that using nu this way does not load your config file by default. Someone else suggested doing `nu --config $nu.config-path test.nu`. Maybe it was someone else who asked?

**Comment by agavrilov** [comment]
@fdncred Yes, indeed we discussed it in #12490. The difference is that I'm doing it from PowerShell now. The workaround with `--config $nu.config-path` is not really workable as in PowerShell `$nu.config-path` is not available. It makes 'nu script.nu' to be broken on light terminal

**Comment by fdncred** [comment]
If you're in powershell then you'll probably have to provide the full path to the config file.

**Comment by agavrilov** [comment]
@fdncred Well, yes, I could specify a full path to the config file, but it is not very ergonomic, don't you agree? Nu scripts should work out-of-the-box on light background terminals without any additional parameters. 

BTW, it looks like 'nu' reads 'env.nu' before executing any scripts. Would it be possible to provide some kind of indication to Nu there what kind of the terminal backround used?

**Comment by fdncred** [comment]
This is how it works now https://gist.github.com/fdncred/b87b784f04984dc31a150baed9ad2447/

I'm not saying this is perfect, just how it works based on me tracing through the code at some point.

Nushell cannot detect background color because it's inconsistently supported in terminal emulators. It wouldn't be too tricky to create a script that launches nushell with the appropriate config file.

I'm not saying we couldn't or shouldn't change things. It's a bit wonky to me how it works. We have the option with -n to not load any config files, so maybe they should be loaded all the time except when that parameter is given? I'm not sure. It'll take more than me to decide.
